pygmt/datasets/moon_relief.py

```diff
@@ -0,0 +1,93 @@
+"""
+Function to download the Moon relief datasets from the GMT data server,
+and load as :class:`xarray.DataArray`.
+
+The grids are available in various resolutions.
+"""
+from pygmt.datasets.load_remote_dataset import _load_remote_dataset
+from pygmt.helpers import kwargs_to_strings
+
+__doctest_skip__ = ["load_moon_relief"]
+
+
+@kwargs_to_strings(region="sequence")
+def load_moon_relief(resolution="01d", region=None, registration=None):
+    r"""
+    Load Moon relief in various resolutions.
+
+    The grids are downloaded to a user data directory
+    (usually ``~/.gmt/server/moon/moon_relief/``) the first time you invoke
+    this function. Afterwards, it will load the grid from the data directory.
+    So you'll need an internet connection the first time around.
+
+    These grids can also be accessed by passing in the file name
+    **@moon_relief**\_\ *res*\[_\ *reg*] to any grid plotting/processing
+    function. *res* is the grid resolution (see below), and *reg* is grid
```

Comment:
```suggestion
    function. *res* is the grid resolution (see below), and *reg* is the grid
```

Reply:
I submitted an PR to fix this for the documentations of the other datasets.